src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DataSyncTests.ps1

```diff
@@ -0,0 +1,614 @@
+		Assert-Null $sm2.MemberDatabasePassword
+		Assert-Null $sm2.SyncAgentId
+		Assert-Null $sm2.SqlServerDatabaseId
+    }
+    finally
+    {
+        Remove-ResourceGroupForTest $rg
+    }
+}
+
+<# 
+    .SYNOPSIS
+    Tests for refreshing and getting a sync member schema
+#>
+function Test-RefreshAndGetSyncMemberSchema
+{
+	# Setup
+	$testSuffix = 9015
+	$rg = Create-ResourceGroupForTest
+	$server = Create-ServerForTest $rg "12.0" "West US 2"
+	$serverName = "$($server.ServerName).sqltest-eg1.mscds.com"
+	$credential = Get-ServerCredential
+	$databaseName1 = Get-DatabaseName
+	$db1 = New-AzureRmSqlDatabase -ResourceGroupName $rg.ResourceGroupName -ServerName $server.ServerName -DatabaseName $databaseName1
+	$databaseName2 = Get-DatabaseName
+	$db2 = New-AzureRmSqlDatabase -ResourceGroupName $rg.ResourceGroupName -ServerName $server.ServerName -DatabaseName $databaseName2
+	$syncDatabaseName = Get-DatabaseName
+	$syncdb = New-AzureRmSqlDatabase -ResourceGroupName $rg.ResourceGroupName -ServerName $server.ServerName -DatabaseName $syncDatabaseName
+	
+	$params = Get-SqlSyncGroupTestEnvironmentParameters $testSuffix
+    $sgName = Get-SyncGroupName
+	$sg = New-AzureRmSqlSyncGroup -ResourceGroupName $rg.ResourceGroupName -ServerName $server.ServerName `
+				-DatabaseName $databaseName1 -SyncGroupName $sgName -IntervalInSeconds $params.intervalInSeconds `
+				-ConflictResolutionPolicy $params.conflictResolutionPolicy -SyncDatabaseName $syncDatabaseName -SyncDatabaseServerName `
+				$server.ServerName -SyncDatabaseResourceGroupName $rg.ResourceGroupName -DatabaseCredential $credential
+	# Create a sync member 
+	$smParams = Get-SqlSyncMemberTestEnvironmentParameters $testSuffix
+    $smName = Get-SyncMemberName
+	$sm1 = New-AzureRmSqlSyncMember -ServerName $server.ServerName -ResourceGroupName $rg.ResourceGroupName `
+        -DatabaseName $databaseName1 -SyncGroupName $sgName -SyncMemberName $smName `
+		-SyncDirection $smParams.syncDirection -MemberDatabaseType $smParams.databaseType -MemberDatabaseName $databaseName2 `
+		-MemberServerName $serverName -MemberDatabaseCredential $credential
+    try
+    {
+		# Refresh member schema
+		Invoke-AzureRmSqlSyncSchemaRefresh -ResourceGroupName $rg.ResourceGroupName -ServerName $server.ServerName `
+            -DatabaseName $databaseName1 -SyncGroupName $sgName -SyncMemberName $smName 
+
+		# Get member schema
+		$schema = Get-AzureRmSqlSyncSchema -ResourceGroupName $rg.ResourceGroupName -ServerName $server.ServerName `
+            -DatabaseName $databaseName1 -SyncGroupName $sgName -SyncMemberName $smName 
+		Assert-NotNull $schema
+    }
+    finally
+    {    
+        Remove-ResourceGroupForTest $rg
+    }
+}
+
+<# 
+    .SYNOPSIS
+    Tests for removing a sync member
+#>
+function Test-RemoveSyncMember
+{
+	# Setup
+	$testSuffix = 9016
+	$rg = Create-ResourceGroupForTest
+	$server = Create-ServerForTest $rg "12.0" "West US 2"
+	$serverName = "$($server.ServerName).sqltest-eg1.mscds.com"
```

Comment:
Hardcoding the dns suffix is bad, this test will only work when run against stage environment. 

@shahabhijeet is there a good way to get SQL DNS suffix from test environment so that it works on both stage and prod depending on the login?